src/components/People.js

```diff
@@ -0,0 +1,85 @@
+import React, { Component } from "react";
+import axios from "axios";
+
+class People extends Component {
+  constructor() {
+    super();
+    this.state = {
+      isError: false,
+      searchPeople: "",
+      people: {},
```

Comment:
`people` should almost definitely be called `person`, since it's always just one person. (Witness the weirdness of saying, `people` should have the same value as `person`.)